neurodamus/core/configuration.py

```diff
@@ -332,7 +332,7 @@ def _init_hoc_config_objs(cls):
             logging.info("User-defined RNG base seed %s", parsed_run.valueOf("BaseSeed"))
 
         if cls._simconf.coreNeuronUsed():
-            cls.coreneuron = h.CoreConfig(cls.output_root)
+            cls.coreneuron = True
```

Comment:
Per comment above, if we dont want the var to hold the object maybe we we can delete it